### DISCUSSION TITLE
VSCode Copilot extension is opening too many files

### DISCUSSION THREAD

**Discussion by huww98** [community/community#54683] [Copilot Conversations]
### Select Topic Area

Bug

### Body

As I described in https://github.com/clangd/vscode-clangd/issues/480, after upgrading "GitHub Copilot" extension to 1.85.75, clangd start to generate a lot of PCH files, consuming all system resources.

**Comment by Akash1134** [comment]
Hi @huww98 ,

We have reported this issue internally and our team will look into it. Thanks!

**Comment by JustinPierce** [comment]
This might be related, but Copilot is also blowing up the TS/JS service in my VSCode. Following the suggestion here, downgrading the extension to 1.84.51 resolves the issue.

**Comment by huww98** [comment]
The latest version seems OK now.
